lib/fprotect/fprotect_acl.c

```diff
@@ -8,17 +8,34 @@
 #include <hal/nrf_ficr.h>
 #include <errno.h>
 #include <sys/__assert.h>
+#include <kernel.h>
 
-static uint32_t region_idx;
+static int find_region_idx(uint32_t *region_idx)
```

Comment:
This function(correct me if I'm wrong) finds non locked regions right?
so maybe name it something with regards to that?

Reply:
I settled on find_free_region()